src/Scripting/CoreTest/TestCompilationFactory.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.VisualBasic;
```

Comment:
I'm not sure why this is in the Core project at all - it's clearly specific to C# and it's only consumed from C#.  It would probably make more sense to move this type to the C# project and then create a corresponding one in the VB project.